src/azure-cli/azure/cli/command_modules/storage/operations/blob.py

```diff
@@ -91,13 +91,21 @@ def update_container_rm(cmd, instance, metadata=None, public_access=None,
                         default_encryption_scope=None, deny_encryption_scope_override=None,
                         enable_nfs_v3_root_squash=None, enable_nfs_v3_all_squash=None):
     BlobContainer = cmd.get_models('BlobContainer', resource_type=ResourceType.MGMT_STORAGE)
+
+    # # TODO will remove warning as well as the parameters in November 2023
+    if default_encryption_scope is not None:
+        logger.warning('--default-encryption-scope cannot be updated after container is created. '
+                       'When it is provided, it will be ignored by the server. '
+                       'This parameter will be removed for the update command in November 2023.')
+    if deny_encryption_scope_override is not None:
+        logger.warning('--deny-encryption-scope-override cannot be updated after container is created. '
+                       'When it is provided, it will be ignored by the server. '
+                       'This parameter will be removed for the update command in November 2023.')
     blob_container = BlobContainer(
         metadata=metadata if metadata is not None else instance.metadata,
         public_access=public_access if public_access is not None else instance.public_access,
-        default_encryption_scope=default_encryption_scope
-        if default_encryption_scope is not None else instance.default_encryption_scope,
-        deny_encryption_scope_override=deny_encryption_scope_override
-        if deny_encryption_scope_override is not None else instance.deny_encryption_scope_override,
+        default_encryption_scope=instance.default_encryption_scope,
+        deny_encryption_scope_override=instance.deny_encryption_scope_override,
```

Comment:
Better to keep the original behavior until `CLI 2.54.0`. Leave it to be ignored by service side, not ignore from client side

Reply:
Got it, make sense.